packages/nx/src/hasher/native-task-hasher-impl.spec.ts

```diff
@@ -149,9 +154,9 @@ describe('native task hasher', () => {
             "AllExternalDependencies": "3244421341483603138",
             "env:NONEXISTENTENV": "3244421341483603138",
             "env:TESTENV": "11441948532827618368",
-            "parent:ProjectConfiguration": "15828052557461792163",
+            "parent:ProjectConfiguration": "4131510303084753861",
             "parent:TsConfig": "2264969541778889434",
-            "parent:{projectRoot}/**/*": "3244421341483603138",
+            "parent:{projectRoot}/**/*": "15295586939211629225",
```

Comment:
These hashes changed because I fixed the root in the graph, which was incorrect.